packages/desktop-client/src/components/util/GenericInput.js

```diff
@@ -149,6 +150,22 @@ export default function GenericInput({
       }
       break;
 
+    case 'boolean':
+      switch (field) {
+        case 'cleared':
+          content = (
+            <Checkbox
+              checked={value}
+              value={value}
+              onChange={e => onChange(!value)}
+            />
+          );
+          break;
+
+        default:
+      }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why would we need a `switch` case here? IMO we can return the checkbox for all fields. And if sometime in the far future we need to have a different functionality for some other field - we can add it then. But until that time comes we can keep the code simple.

```suggestion
          content = (
            <Checkbox
              checked={value}
              value={value}
              onChange={e => onChange(!value)}
            />
          );
      }
```

Reply:
That's a great point.  I'll make the change.